Add partiql-planner and partiql-parser Maven publishing plugin v0.12.1 (#1216)
partiql/partiql-lang-kotlin: CHANGELOG.md

-->

## [0.12.1] - 2023-09-19

> ### Added

Reviewer: Same as other PRs.

> - Fixes build failure for version `0.12.0` by publishing `partiql-plan` and `partiql-parser` as an independent artifact. Please note that `partiql-plan` is experimental.

Reviewer: ```suggestion
- Fixes build failure for version `0.12.0` by publishing `partiql-plan` and `partiql-parser` as independent artifacts. Please note that `partiql-plan` and `partiql-parser` are experimental.
```
